src/helpers/period.coffee

```diff
@@ -0,0 +1,7 @@
+_ = require 'underscore'
+
+PeriodHelper =
+  getOrder: (period) ->
+    parseInt(period.name)
```

Comment:
Oh, okay, is there a list of valid period names/patterns somewhere?  I guess this should just plan for all cases?